instagram-page-react/src/pages/profilePage/molecules/profileTabbar/ProfileTabbar.js

```diff
@@ -0,0 +1,48 @@
+import React, {useCallback, useState} from 'react';
+
+// Lodash
+import _map from 'lodash/map';
+
+// Styles
+import styles from './profileTabbar.module.css'
+
+// Constants
+import PROFILE_TABBAR_CONFIG from "./constants/profileTabbar.configs";
+
+// Components
+import ProfileTabbarTab from "./atoms/profileTabbarTab";
+
+function ProfileTabbar() {
+    const [selectedTabId, setSelectedTabId] = useState("POSTS");
+
+    const handleProfileTabbarClick = (tabID) => {
+        setSelectedTabId(tabID);
+    }
+
+    const renderProfileTabbar = useCallback((tab) => {
```

Comment:
This is not required, we are not passing this function to a child, we are only internally consuming it in this component